src/components/SearchList.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
```

Comment:
The `SearchScreen` makes use of the `SearchList` and `SearchMap` components, which are below.  The actual `SearchScreen` is where we navigate to after logging in, and it will display the appropriate component based on the current state.